.github/workflows/bazel.yml

```diff
@@ -119,8 +128,43 @@ jobs:
         uses: browser-actions/setup-edge@latest
         with:
           edge-version: ${{ inputs.browser-version || 'stable' }}
+      - name: Get Nightly Version
+        if: inputs.get-nightly-version == true
+        run: |
+          LATEST_TAG=$(git describe --tags --abbrev=0 --exclude="nightly")
+          RELEASE_VERSION=$(echo "$LATEST_TAG" | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+')
+          NEW_VERSION=$(echo "$RELEASE_VERSION" | awk -F. '{print $1"."$2+1".0"}')
+          NIGHTLY_VERSION="$NEW_VERSION.nightly.$(date +%Y%m%d)"
+          echo "NIGHTLY_VERSION=$NIGHTLY_VERSION" >> $GITHUB_ENV
+      - name: Update Version
+        if: inputs.language-release-setup == 'python'
+        run: |
+          ./scripts/github-actions/update-version.sh ${{ github.env.NIGHTLY_VERSION }}
+      - name: Install Python
+        if: inputs.language-release-setup == 'python'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
```

Comment:
Can we bump this in the same way it is done for Ruby? 
https://github.com/SeleniumHQ/selenium/blob/dd10ff8f89df8ee1dd02664f0832042d7f50458f/.github/workflows/nightly.yml#L36C1-L39

Reply:
removed the extra stuff of getting tags from git to simplify, did try to get to work with a bazel function but the main issue I ran into was ruby never referenced the version in the build file which python did multiple times. Not well versed in bazel either which made it more difficult lol I assume it can be done but I would defer it to someone more familiar with bazel